pages/webview/content_page.py

```diff
@@ -23,14 +22,31 @@ def books_containing(self):
     class BooksContaining(Region):
         _root_locator = (By.CLASS_NAME, 'booksContaining')
         _book_list_locator = (By.CSS_SELECTOR, 'div > ul > li')
+        _nav_title_locator = (By.CSS_SELECTOR, 'span.title')
+
+        @property
+        def nav_title(self):
+            return self.find_element(*self._nav_title_locator)
 
         @property
+        @retry_stale_element_reference_exception
         def book_list(self):
             return [self.Book(self.page, el) for el in self.find_elements(*self._book_list_locator)]
 
+        @property
+        @retry_stale_element_reference_exception
+        def go_to_book_link(self):
```

Comment:
This is called `go_to_book_link` but returns a list of links? I don't think this is necessary. In your test you should iterate over the book list and assert that there is a `got_to_book_link`.

```
book_list = content.books_containing.book_list

for book in book_list:
    assert book.go_to_book_link.is_displayed()